packages/angular/src/jsonforms.service.ts

```diff
@@ -19,11 +30,11 @@ export class JsonFormsAngularService {
         this.state = new BehaviorSubject({ jsonforms: this._state });
     }
 
-    subscribe(observer: PartialObserver<JsonFormsState>): Subscription {
+    get $state(): Observable<JsonFormsState> {
```

Comment:
I suggest to keep the subscribe method but mark it as deprecated.
@sdirix what do you think?

Reply:
Besides some alpha releases this was not yet made public so I would be fine with changing this now without deprecation.

Reply:
agreed,